android-sdk/src/main/java/com/checkout/android_sdk/logging/FramesLoggingEventDataProvider.kt

```diff
@@ -1,18 +1,28 @@
 package com.checkout.android_sdk.logging
+import android.content.Context
+import com.checkout.android_sdk.Utils.CheckoutTheme
 import com.checkout.android_sdk.Utils.Environment
+import com.checkout.android_sdk.Utils.filterNotNullValues
 import com.checkout.eventlogger.domain.model.MonitoringLevel
 import java.util.Locale
 
 object FramesLoggingEventDataProvider {
 
-    fun logPaymentFormPresentedEvent(): FramesLoggingEvent {
+    fun logPaymentFormPresentedEvent(context: Context): FramesLoggingEvent {
```

Comment:
Can you not just pass CheckoutTheme into logPaymentFormPresentedEvent as param instead of Context ?

Reply:
Yes, we can pass the CheckoutTheme as well.